core/sky_model.py

```diff
@@ -672,7 +672,7 @@ def run_make_sky_model(
             else:
                 chan_opt += f"--channels {g[0]}~{g[-1]+1}"
 
-        cmd = f"time python vsim.py sky-model {sky_model} --local --nside {nside} {chan_opt}"
+        cmd = f"time python vsim.py sky-model {sky_model} --local --nside {nside} --eor-label '{label}' {chan_opt}"
```

Comment:
Here, the flag is `--eor-label` as opposed to `--label` as in line 636

Reply:
Ah, good catch!